src/services/userService.js

```diff
@@ -30,4 +30,9 @@ const signin = async (email, password) => {
   return jwtToken;
 };
 
-module.exports = { signup, signin };
+const getUserById = async (id) => {
+  const userId = await userDao.getUserById(id)
+  return userId
```

Comment:
userId보다는 가져오는 데이터가 user이므로 변수명을 아래와 같이 고치면 좋을 것 같네요.

```suggestion
  const user = await userDao.getUserById(id)
  return user
```